cookbooks/travis_ci_connie/metadata.rb

```diff
@@ -8,7 +8,6 @@
 source_url 'https://github.com/travis-ci/packer-templates'
 issues_url 'https://github.com/travis-ci/packer-templates/issues'
 
-depends 'nodejs'
```

Comment:
This cookbook dependency is no longer necessary, since the equivalent recipe(s) are now provided via the `travis_build_environment` cookbook.